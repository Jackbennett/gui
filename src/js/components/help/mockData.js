export const helpProps = {
  docsVersion: '',
  findLocalIpAddress: jest.fn,
  isHosted: false,
  isEnterprise: false,
  menderDebPackageVersion: 'master',
  menderVersion: 'master',
  menderArtifactVersion: 'master',
  org: {
    tenant_token: ''
  }
};
