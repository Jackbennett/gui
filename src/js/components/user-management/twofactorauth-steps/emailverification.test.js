import React from 'react';
import { render } from '@testing-library/react';
import EmailVerification from './emailverification';
import { undefineds } from '../../../../../tests/mockData';

describe('EmailVerification Component', () => {
  it('renders correctly', async () => {
    const { baseElement } = render(<EmailVerification verifyEmailComplete={jest.fn} verifyEmailStart={jest.fn} />);
    const view = baseElement.firstChild;
    expect(view).toMatchSnapshot();
    expect(view).toEqual(expect.not.stringMatching(undefineds));
  });
});
