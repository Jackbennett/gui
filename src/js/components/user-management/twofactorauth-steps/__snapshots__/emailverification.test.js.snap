// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmailVerification Component renders correctly 1`] = `
<div>
  <div
    class="margin-top"
  >
    Please verify your email address first, to enable Two Factor Authentication.
    <br />
    Once you click the button below we will send you an email with a confirmation code in it and a confirmation field will appear below. Enter the confirmation code in the confirmation field to complete the verification process.
    <div
      class="flexbox"
      style="align-items: center;"
    >
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
        style="margin-top: 20px; margin-right: 30px;"
        tabindex="0"
        type="button"
      >
        <span
          class="MuiButton-label"
        >
          Verify your email address
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
      <div
        class="loaderContainer shrunk"
      >
        <div
          class="loader"
        >
          <span
            class="dot dot_1"
          />
          <span
            class="dot dot_2"
          />
          <span
            class="dot dot_3"
          />
          <span
            class="dot dot_4"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
